test/paymentsSpec.js

```diff
@@ -253,6 +253,21 @@ describe('Payments', function () {
     }
   })
 
+  it('relays a rejection', function * () {
```

Comment:
Shouldn't it relay the actual message the destination transfer was rejected with?